problem2.py

```diff
@@ -1 +1,22 @@
+###so it takes the 3 side mesures and it adds all the combinations together if any 1 of them adds up to 2 of the others it is a triangle###
 
+print("welcome to the triangle checker")
+adj = float(input("what is the lenght of the adjacent side: "))
+opp = float(input("what is the lenght of the opposite side: "))
+hyp = float(input("what is the lenght of the hypotenouse: "))
+###this takes the 3 mesurments down###
+
+if (adj + opp) >= hyp : 
+  print("this is a triangle")
+elif (hyp + opp) >= adj : 
+  print("this is a triangle")
+elif (adj + hyp) >= opp : 
+  print("this is a triangle")
```

Comment:
if-elif-else statement block is functional, however, use of `or` to logically combine these together is ideal